dev-requirements.txt

```diff
@@ -17,6 +17,7 @@ isort==5.10.1
 # Running tests
 pytest
 pytest-sphinx
+datasets  # We have a test that needs this.
```

Comment:
To keep our dev-dependencies light, that test could be put into `tests/integrations/datasets` instead.

Reply:
Hmm. It's a test for `MappedSequence`. You have to special case when the sequence is a `Dataset`, because `Dataset`s look like sequences, but they are not. It looks and quacks like a duck, but it's actually a platypus.

I'll do it anyways.